### ISSUE TITLE
xcode light?

### ISSUE THREAD

**Issue by kujohn** [ghashy/helix-xcode-theme#2]
Any plans to port the light theme as well?

**Comment by 39george** [comment]
@kujohn,
Yes, I do have plans to port the light theme as well. However, the current version I have is too rough and dirty to upload to GitHub.
